src/components/core/geoChart/geoChartDataSource.ts

```diff
@@ -74,6 +96,7 @@ function transformPushpinDataSource(
                         coordinates: [lng, lat], // Mapbox requires number[]
                     },
                     properties: {
+                        drillableProps: getDrillableProps(dataSourceProps, index),
```

Comment:
Intead of create N `drillableProps ` for N pins, we should move this creation to `handleMapClick` so that `drillableProps ` only is created once user click on specific pin.
This could improve perf of geo rendering a bit.

Reply:
Fixed. 
`drillableProps` is called only when pin is clicked